Cython/Compiler/Nodes.py

```diff
@@ -5191,6 +5191,10 @@ def analyse_declarations(self, env):
             scope.doc = embed_position(self.pos, self.doc)
 
         if has_body:
+            properties = {}
+            for stat in getattr(self.body, 'stats', []):
```

Comment:
Calling `getattr()` always has a code smell. Is the problem here that `self.body` might have been wrapped? Because if that is the case, then it seems wrong to ignore the body.

Reply:
I can't immediately find why was there (it seems to pass all tests with "prop" in the name without it). I've tried removing it (and will either leave it removed or insert an appropriate comment if it is needed)